File: greenyun2/217_Contains_Duplicate.js
var containsDuplicate = function (nums) {
  /**
   * new Set 사용시 중복값이 제거 됩니다.
   * 원본 배열과 길이가 다르다면 중복이 있으므로 true를 반환합니다.
   * 원본 배열과 길이가 같다면 false를 반환합니다.
   *  */
  return [...new Set(nums)].length !== nums.length;
};

File: ynot0117/217_Contains_Duplicate.cpp
class Solution {
public:
    bool containsDuplicate(vector<int>& nums) {
        sort(nums.begin(), nums.end());

        int before = nums[0];
        for(int index = 1; index < nums.size(); index++)
        {
            int now = nums[index];
            if (now == before)
                return true;
            before = now;
        }
        return false;
    }
};